src/src/AppContext.js

```diff
@@ -63,6 +66,12 @@ function AppProvider({ children }) {
     await loadMyProfile();
   }
 
+  async function addNewRepository(name, description) {
+    createRepository(name, description);
+    await sleep(3000);
```

Comment:
Not necessary to fix:
could we await the `createRepository` call instead.
I know we do these "pretend we work hard" waits in elsewhere too, but I think we should get rid of them.
Let us discuss after christmas.